### PR TITLE
New lesson: Career - Working and Collaborating Remotely

### LINKED ISSUE #1295: Create new lesson on Working & Collaborating Remotely
It was already a useful skill but now is vital in these times.

## Requirements
- Use the [lesson template](https://github.com/Techtonica/curriculum/tree/master/_templates) to create a lesson
- The new lesson should be placed in the `/career` folder
- For activities/practice on this, have the participants try out popular free screen sharing software to pair program on some code to see the challenges

### PR DESCRIPTION
**Pull request by datadaveshin**
Resolves #1295 from the Hackathon
Previous PR was #1321 

Replacing with this PR due to some git issues

Review ended with revising the "Challenges" section with something more explicit and shorter.

### CODE REVIEW

File: career/working-and-collaborating-remotely.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  - Use tools like Pomodoros and timers
+  - Make your own Kanban board for micromanaging smaller parts of your tickets
+  - Avoid "busy work" and focus on your assignments
+  - Turn off unimportant notifications and leave the TV for "life" hours
+  - Block time to get in the coding zone, list your other todos that aren't immediate and block time for those separately
+  - A proper diet - one can keep a journal and monitor their energy levels according to what they eat to see what's best  
+  - Plan meals - make a specific lunch time as opposed to snacking
+  - Exercise - many tech jobs are completely sedentary, exercise can boost productivity and morale
+
+* **Ergonomics** - Just like any other working situation, it is vitally important to maintain good ergonomics while working remotely.  However, the responsibility will likely be on you, the maintainer of your working space, to take the lead on either assessing your work station, or, working with a supervisor or HR to have someone from your business assess your workstation
```
Comment (vegetabill) [inline career/working-and-collaborating-remotely.md:81]: Can you link a part of this to our existing ergo lesson (using a relative link):

https://github.com/Techtonica/curriculum/blob/master/onboarding/ergonomics.md

File: career/working-and-collaborating-remotely.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  - Ask your supervisor or team lead what channels you should subscribe to on the company’s chat program
+  - Identify, through looking at various channels, where certain technical and non-technical questions should be directed to. This may be hr@yourcompany.com for some questions, or a slack message for the devops channel for others
+  - Build relationships with your team - see the section below for more info
+
+#### Professionalism and Growth
+* **Being visible for professional growth** - The old adage, “out of sight, out of mind” can easily apply to remote working situations, and it can be hard to make an impression when you are not in the same proximity as your co-workers.  Making an effort to bridge the distance will not only make it easier to work with your team, it will help you make connections you will utilize throughout your career.
+  - Let people know what you are working on - you don’t need to give everyone a long list, but be "vocal" online about your projects and share any roadblocks you might have. With that said, be self-sufficient and try to solve hiccups on your own before consulting the team, and, make sure to help others resolve their blockers too
```
Comment (vegetabill) [inline career/working-and-collaborating-remotely.md:120]: Maybe suggest sharing your successes, e.g. a screenshot or screencap of a feature completed or problem solved?

File: career/working-and-collaborating-remotely.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+  - For a more complete list see this list from [designmodo](https://designmodo.com/code-collaboration/)
+* **Asking questions** - In a remote environment, you can’t lean over and ask questions to the person sitting next to you or in the breakroom. To help you get the help you need you should:
+  - Ask your supervisor or team lead what channels you should subscribe to on the company’s chat program
+  - Identify, through looking at various channels, where certain technical and non-technical questions should be directed to. This may be hr@yourcompany.com for some questions, or a slack message for the devops channel for others
+  - Build relationships with your team - see the section below for more info
+
+#### Professionalism and Growth
+* **Being visible for professional growth** - The old adage, “out of sight, out of mind” can easily apply to remote working situations, and it can be hard to make an impression when you are not in the same proximity as your co-workers.  Making an effort to bridge the distance will not only make it easier to work with your team, it will help you make connections you will utilize throughout your career.
```
Comment (vegetabill) [inline career/working-and-collaborating-remotely.md:119]: This is a great point. My brother is full-time remote and he said making time to be visible in Slack is an important part of his day.

File: career/working-and-collaborating-remotely.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+- **They are giving me a laptop, now I don't need to buy a new one** - If you are thinking that now you don’t have to buy a new computer to contribute to your non-profit or build your Etsy shop, think again. Your work laptop is your company’s property, it should be used for work only. Moreover, some companies may install software that monitors your work.
+
+### Guided Practice
+In groups of 2 or 3, set up a remote pairing environment using only Slack (or another online tool) to communicate to simulate a remote collaboration. For this, an easy to set up free online pairing system is [CodeSandbox](https://codesandbox.io/) which only needs GitHub authentication to pair. See the instructions for pairing [here](https://www.ma-no.org/en/software/development/10-collaborative-coding-tools-for-remote-pair-programming), along with other choices. While still communicating online, pick a small project, hack away and see if you can complete it in the allotted time.
```
Comment (vegetabill) [inline career/working-and-collaborating-remotely.md:147]: Can you specifically mention to pair on one of the "daily code challenges"

File: career/working-and-collaborating-remotely.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+  - A dedicated area(s) for where you do your work
+  - A set of rituals that are done before and after work - such as getting dressed as if going into the office and walking around the block for a "commute", before sitting down at the work station
+  - A separate computer or account to separate work from personal computer time
+  - A discussion about expectations with family or roommates regarding your work time, and a plan as to when and how to interact with others that may share the same household
+- **Time management and productivity** - Another struggle closely related to work/life balance is time management. This  problem arises because as opposed to being around others in a traditional office, there are no social queues for break and lunch times and an environment where you see everyone quiet and working. Tips:
+  - Have a clean and uncluttered work area
+  - Set a specific weekly and daily schedule for work time
+  - Plan your day with a task list
+  - Use tools like Pomodoros and timers
```
Comment (vegetabill) [inline career/working-and-collaborating-remotely.md:72]: Can you link to some faves?

For Pomodoro, I've liked https://apps.apple.com/us/app/be-focused-focus-timer/id973134470?mt=12

File: career/working-and-collaborating-remotely.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+- Watch [10 tips for productive remote work](https://www.youtube.com/watch?v=0JbeJgcRPMM)
+
+### Lesson
+**Remote work** is defined as working offsite from a traditional office. **Remote collaboration** is defined as working with at least one person who is physically at another site, independent of whether the sites are within a traditional office setting. Both present both pros and cons. For example, some pros would be that remote collaboration saves both the employees and company the costs of travel time, and remote work saves the employee the costs of commuting and the company the costs of office space. On the con side, both remote collaboration and remote work lose the benefits of interpersonal communication that can take place when everyone involved are within proximity. Challenges on the con side are exemplified in the first minute and a half of this video on the [remote work revolution](https://www.youtube.com/watch?v=wUTd35A7I6c), where the speaker notes the following quote:
+
+> 1995: The information superhighway will mean anyone can do anything from anywhere
+>
+> 2015: must be willing to relocate to San Francisco
```
Comment (vegetabill) [inline career/working-and-collaborating-remotely.md:58]: lulz

File: career/working-and-collaborating-remotely.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+  1. Start working on the challenges
+  1. When the first person has completed a challenge, have them move the link into the "Done" column, and then pair program with one of the other people that are still working on their challenge until completed
+  1. If the team has a total of two people, start another round picking two new challenges from the "Todo" column. Otherwise, if there is a third person, have the two that just wrapped up the second code challenge together team up with the third person to complete the round. Then again, pick out 3 new challenges, and repeat
+
+See how many challenges you can get through, and save the sheet in case you have time in the future to solve more challenges. Seeing the list grow will build your confidence!
+
+### Check for Understanding
+Go onto Youtube or search for blog posts about the “pros and cons” of remote work, the etiquette of communicating online, or professional growth for remote workers. Write a short summary of what you found to be a surprise or misconception, or on the other hand, solidified your beliefs about the subject.
+
+### Supplemental Materials
```
Comment (vegetabill) [inline career/working-and-collaborating-remotely.md:168]: DHH's blog might be good to include. IIRC his book remote was made up mostly of blog content.

Related, here are some cool setups:
https://m.signalvnoise.com/remote-working-the-home-office-desks-of-basecamp/

**Comment by datadaveshin** [comment]
Thanks so much for the review - I'll try to get the edits in over the weekend
Also thanks for timer suggestion, I want to try it!

File: career/working-and-collaborating-remotely.md (hunk 2 above)
Comment (datadaveshin) [reply career/working-and-collaborating-remotely.md:120]: Ok I fit it in

File: career/working-and-collaborating-remotely.md (hunk 5 above)
Comment (datadaveshin) [reply career/working-and-collaborating-remotely.md:72]: Sure, I put in AS Timer for devises, and marinartimer which is shared over the web

File: career/working-and-collaborating-remotely.md (hunk 4 above)
Comment (datadaveshin) [reply career/working-and-collaborating-remotely.md:147]: Yep

File: career/working-and-collaborating-remotely.md (hunk 7 above)
Comment (datadaveshin) [reply career/working-and-collaborating-remotely.md:168]: Added the signal to noise blog if that's what you meant, I wasn't sure. Also included a list of blogs that deal with remote work

File: career/working-and-collaborating-remotely.md (hunk 1 above)
Comment (datadaveshin) [reply career/working-and-collaborating-remotely.md:81]: Done, see below

File: career/working-and-collaborating-remotely.md (hunk 3 above)
Comment (datadaveshin) [reply career/working-and-collaborating-remotely.md:120]: Done, see below

File: career/working-and-collaborating-remotely.md [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+- **They are giving me a laptop, now I don't need to buy a new one** - If you are thinking that now you don’t have to buy a new computer to contribute to your non-profit or build your Etsy shop, think again. Your work laptop is your company’s property, it should be used for work only. Moreover, some companies may install software that monitors your work.
+
+### Guided Practice
+In groups of 2 or 3, set up a remote pairing environment using only Slack (or another online tool) to communicate to simulate a remote collaboration. For this, an easy to set up free online pairing system is [CodeSandbox](https://codesandbox.io/) which only needs GitHub authentication to pair. See the instructions for pairing [here](https://www.ma-no.org/en/software/development/10-collaborative-coding-tools-for-remote-pair-programming), along with other choices. While still communicating online, work on one of the Techtonica daily code challenges. If you finish it, and still have time remaining to continue, pair on one of the daily code challenges that one of you were stuck on previously.
```
Comment (datadaveshin) [inline career/working-and-collaborating-remotely.md:148]: Edited to have apprentices work on daily code challenges as suggested

File: career/working-and-collaborating-remotely.md [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+  - A separate computer or account to separate work from personal computer time
+  - A discussion about expectations with family or roommates regarding your work time, and a plan as to when and how to interact with others that may share the same household
+- **Time management and productivity** - Another struggle closely related to work/life balance is time management. This  problem arises because as opposed to being around others in a traditional office, there are no social queues for break and lunch times and an environment where you see everyone quiet and working. Tips:
+  - Have a clean and uncluttered work area
+  - Set a specific weekly and daily schedule for work time
+  - Plan your day with a task list
+  - Use tools like Pomodoros and timers. Some suggestions for your laptops/devices are the simple [AS Timer](https://apps.apple.com/us/app/as-timer/id512464723) and the [Be Focused](https://apps.apple.com/us/app/be-focused-focus-timer/id973134470?mt=12) timer, which has more features. For sharing online for teamwork, the web-based [MarinaraTimer](https://www.marinaratimer.com/) is also quite handy
```
Comment (datadaveshin) [inline career/working-and-collaborating-remotely.md:72]: Added links to both native and web-based timers, including one suggested by Bill

File: career/working-and-collaborating-remotely.md [hunk 10]
```diff
@@ -0,0 +1,174 @@
...
+  - Build relationships with your team - see the section below for more info
+
+#### Professionalism and Growth
+* **Being visible for professional growth** - The old adage, “out of sight, out of mind” can easily apply to remote working situations, and it can be hard to make an impression when you are not in the same proximity as your co-workers.  Making an effort to bridge the distance will not only make it easier to work with your team, it will help you make connections you will utilize throughout your career.
+  - Let people know what you are working on - you don’t need to give everyone a long list, but be "vocal" online about your projects and share any roadblocks you might have. Sharing an occasional screenshot or screencap of a feature completed or problem solved is another good way to show progress. With that said, be self-sufficient and try to solve hiccups on your own before consulting the team, and, make sure to help others resolve their blockers too
```
Comment (datadaveshin) [inline career/working-and-collaborating-remotely.md:121]: Added part about sharing screenshots/screen cap

**Pull request merged**